pkg/operator/ceph/cluster/cluster.go

```diff
@@ -595,28 +600,89 @@ func (c *cluster) reportTelemetry() {
 	telemetry.ReportKeyValue(c.context, c.ClusterInfo, telemetry.ExternalModeEnabledKey, strconv.FormatBool(c.Spec.External.Enable))
 }
 
+func (c *cluster) addSettingsToMonDB(clientName string, settings map[string]string) error {
+
+	assimilateConfPath, err := ioutil.TempFile("", fmt.Sprintf("%s.txt", clientName))
```

Comment:
Do we need to append the clientName.txt? Seems like a random temp file name is fine, and it doesn't need the txt extension or specific name.

Reply:
no, we don't need `clientName.txt` or `clientName` random file will work but I have kept the `clientName` in patter for file generation since it is helping in unit tests to fetch the file and test the content of the file.
If we want, I can remove `clientName` to but it looks good in the unit test to check the file content.